agent-chart/values.yaml

```diff
@@ -18,20 +91,28 @@ canary-checker:
     secretKeyRef:
       name: upstream
   db:
-    embedded:
-      persist: true
+    runMigrations: false
+    external:
+      enabled: true
+      create: false
+      secretKeyRef:
+        name: incident-commander-postgres
+        key: DB_URL
   flanksource-ui:
     # Disable canary-checker by default.
     enabled: false
 
 config-db:
+  disablePostgrest: true
   db:
-    enabled: true
-    secretKeyRef:
-      create: true
-    storage: "20Gi"
-    storageClass: ""
-
+    runMigrations: false
+    embedded:
+      persist: false
+    external:
+      enabled: true
+      secretKeyRef:
+        name: incident-commander-postgres
+        key: DB_URL
```

Comment:
I don't think we should default the agentName - it should be empty and then mark as required in schema

Reply:
👍🏽